apps/okreads/browser/src/app/app.component.html

```diff
@@ -23,7 +24,7 @@ <h1>okreads</h1>
     <div class="reading-list-container" data-testing="reading-list-container">
       <h2>
         My Reading List
-        <button mat-icon-button (click)="drawer.close()">
+        <button tabindex="0" mat-icon-button aria-label="Close" (click)="drawer.close()">
```

Comment:
Similarly here, the tabinde is probably unnecessary.

Are you sure tha an `aria-label` of "Close" is sufficient? Close what?